src/main/kotlin/net/theevilreaper/dartpoet/code/CodeWriter.kt

```diff
@@ -259,6 +259,26 @@ class CodeWriter constructor(
         }
     }
 
+    /**
+     * Emits a specific amount of [SPACE] strings to the given [Appendable].
+     * @param amount the amount of spaces which should be applied
+     * @return the instance from the writer
+     */
+    fun emitSpaces(amount: Int = 1) = apply {
+        check(amount > 0) { "The amount can't be negative" }
```

Comment:
Is this correct ?

Reply:
When you mean the `check(amount > 0)` block, yes. It throws a message only when the value is false